components/recurring-contributions/RecurringContributionsCard.js

```diff
@@ -90,46 +85,33 @@ const RecurringContributionsCard = ({
                 <P fontSize="12px" lineHeight="20px" color="black.700">
                   (
                   <FormattedMoneyAmount
-                    amount={contribution.amount.value * 100}
+                    amount={contribution.amount.valueInCents - contribution.platformContributionAmount.valueInCents}
                     currency={contribution.amount.currency}
                     showCurrencyCode={false}
-                    precision={0}
+                    precision="auto"
                     amountStyles={{ fontWeight: 'normal', color: 'black.700' }}
-                  />{' '}
-                  +{' '}
+                  />
+                  {' + '}
                   <FormattedMoneyAmount
-                    amount={contribution.platformContributionAmount.value * 100}
+                    amount={contribution.platformContributionAmount.valueInCents}
```

Comment:
@sbinlondon I replaced all the `value * 100` by `valueInCents`, it's simpler and it avoids an issue with float numbers precision. Let's imagine that someone is contributing $1.10 per month, now please try and type `1.10 * 100` in your console and check the result. What you'll see is the reason why we avoid dealing with floats in general :slightly_smiling_face: 

For more info about that: https://floating-point-gui.de/